### ISSUE TITLE
Preview of cover photo of community is wrong

### ISSUE THREAD

**Issue by lifenautjoe** [OpenbookOrg/openbook-app#200]
We show a different height on the preview for the community when editing it and after it's saved. This is frustrating and we should make it the exact same dimensions.

https://openbook.canny.io/admin/board/feature-requests/p/provide-cover-photo-guidance

**Comment by WouterJanson** [comment]
Changing the default `OBCoverSize` to a `.small` in the `OBCover` constructor would level all covers when it comes to dimensions throughout the app. 
This would already be a quick win, what's the reasoning behind the multiple sizes? Desktop/web and tablet?

**Comment by Komposten** [comment]
> This would already be a quick win, what's the reasoning behind the multiple sizes? Desktop/web and tablet?
They use small size for community covers and normal size for profile covers, so they're both in use in the app.

**Comment by lifenautjoe** [comment]
Yep. It’s to emphasize as much as possible on the distinction between
community and people. It might not look like a big difference but this
together with the profile photo position provide a good cue when navigating
from one to the other, from the context perspective. Color cannot be this
cue as it might be the same / similar color to x community.
